pkg/daemon/rpm-ostree.go

```diff
@@ -253,13 +253,28 @@ func (r *RpmOstreeClient) Rebase(imgURL, osImageContentDir string) (changed bool
 	return
 }
 
+func truncate(input string, length int) string {
+	asRunes := []rune(input)
+
+	if length > len(asRunes) {
+		length = len(asRunes)
+	}
+
+	return string(asRunes[:length])
+}
+
 // runGetOut executes a command, logging it, and return the stdout output.
 func runGetOut(command string, args ...string) ([]byte, error) {
 	glog.Infof("Running captured: %s %s", command, strings.Join(args, " "))
 	cmd := exec.Command(command, args...)
-	rawOut, err := cmd.CombinedOutput()
+	rawOut, err := cmd.Output()
 	if err != nil {
-		return nil, errors.Wrapf(err, "error running %s %s: %s", command, strings.Join(args, " "), string(rawOut))
+		errtext := ""
+		if e, ok := err.(*exec.ExitError); ok {
```

Comment:
On the off chance that this assertion doesn't hold, should we put an else condition to error more gracefully?

Reply:
In that case we'll omit the standard error text, which seems fine?  What would be in the `else`?

I think it's possible to get a different error if we fail to `exec()` instead of the child process itself failing, so it can happen.  But I think there'd just be an extra `: ` in the error message.

But I cleaned this up and added a test for it too.